src/main/scala/rocketchip/DebugTransport.scala

```diff
@@ -159,7 +159,7 @@ class DebugTransportModuleJTAG(debugAddrBits: Int, c: JtagDTMConfig)
   nonzeroResp := stickyNonzeroRespReg | (io.dmi.resp.valid & (io.dmi.resp.bits.resp != UInt(0)))
 
   busyResp.addr  := UInt(0)
-  busyResp.resp  := UInt(0)
+  busyResp.resp  := -1.S(DMIConsts.dmiRespSize.W).asUInt // Generalizing busy to 'all-F'
```

Comment:
I go back and forth on how I like to express all-ones of a certain width... you can also do ```Fill(DMIConsts.dmiRespSize, 1.U)```